ui/components-react/src/components-react/filter-builder/FilterBuilderRuleGroup.tsx

```diff
@@ -61,24 +62,42 @@ export function PropertyFilterBuilderRuleGroupRenderer(
     [path, actions]
   );
 
-  const allowToAddGroup =
-    ruleGroupDepthLimit === undefined || path.length < ruleGroupDepthLimit;
   const { focusedElement, hoveredElement, ...eventHandlers } = React.useContext(
     ActiveRuleGroupContext
   );
 
   const showOperator = group.items.length > 1;
 
   return (
-    <div
+    <Flex
       ref={groupRef}
-      className="rule-group"
+      style={{ alignSelf: "flex-start" }}
+      className="fb-group fb-group-criteria fb-criteria-container"
+      gap="0px"
       data-isactive={
         groupRef.current === focusedElement ||
         groupRef.current === hoveredElement
       }
       {...eventHandlers}
     >
+      {showOperator ? (
+        <PropertyFilterBuilderRuleGroupOperator
+          operator={group.operator}
+          onChange={onOperatorChange}
+        />
+      ) : null}
+      <div className="fb-wrapper">
+        {group.items.map((item) => (
+          <Flex className="fb-row" key={item.id}>
+            <PropertyFilterBuilderGroupOrRule
+              path={path}
+              item={item}
+              onAddRule={handleAddRule}
+              size={size}
+            />
+          </Flex>
+        ))}
+      </div>
       <div className="rule-group-remove-action">
```

Comment:
Shouldn't this be removed now that nesting is not supported?

Reply:
We need both the PropertyFilterBuilderRuleGroupOperator and the PropertyFGilterBuilderGroupOrRule to be rendered here, otherwise nothing is shown. We are still supporting grouping (i.e. 2 or more rules linked by the same oeprator), we're just not supporting nesting i.e. indenting rule(s) under an additional rule group.